pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -142,3 +195,145 @@ func (a *GCPActuator) getImageID(cd *hivev1.ClusterDeployment, logger log.FieldL
 		return "", errors.New(msg)
 	}
 }
+
+// obtainLease uses the Hive MachinePoolNameLease resource to obtain a unique, single character
+// for use in the name of the machine pool. We are severely restricted on name lengths on GCP
+// and effectively have one character of flexibility with the naming convention originating in
+// the installer.
+func (a *GCPActuator) obtainLease(pool *hivev1.MachinePool, cd *hivev1.ClusterDeployment, logger log.FieldLogger) (string, bool, error) {
```

Comment:
Either (1) add details in the function description about what the second return value is or (2) name the return values.

Reply:
Vars named.